eng/common/scripts/Update-DocsMsMetadata.ps1

```diff
@@ -102,7 +114,17 @@ function GetAdjustedReadmeContent($ReadmeContent, $PackageInfo, $PackageMetadata
   if ($codeOwnerArray) {
     Write-Host "Code Owners are $($codeOwnerArray -join ",")"
     $author = $codeOwnerArray[0]
-    $msauthor = $author # This is a placeholder for now. Will change to the right ms alias.
+    try {
+      $msauthor = "$PSScriptRoot/Get-AADIdentityFromGithubUser.ps1" `
+        -TenantId $TenantId `
+        -ClientId $ClientId `
+        -ClientSecret $ClientSecret `
+        -GithubUser $author
+    }
+    catch {
+      Write-Host "No msalias fetching from github username: $author. Use github username for default value."
```

Comment:
I think it's still worth logging the error above this message, because there could be cases where we expect the msalias to be found and it will be easier to debug if we have the exception.